control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -310,6 +310,10 @@ impl VolumeSpec {
         }
         self.target_config.as_ref().map(|c| &c.target.nexus)
     }
+    /// Get `uuid`.
+    pub fn uuid(&self) -> &VolumeId {
```

Comment:
There's already a trait for this: `ResourceUid`.

Reply:
Ahh okay. Will use that.